pages/docs/couple-your-code/couple-your-code-time-step-sizes.md

```diff
@@ -75,6 +75,45 @@ This procedure is independent of whether a serial or a parallel coupling scheme
 For parallel coupling, both solvers run together and everything happens simultaneously in both participants, while for serial coupling, the first participant needs reach the end of the window before the second one can start.
 {% endnote %}
 
+### Possible subcycling pitfall
+
+If you are using very small many time steps in one window, you might see the following error message:
+```
+ERROR: preCICE has detected a difference between its internal time and the time of this participant. This can happen, if you are using very many substeps per time window over multiple time windows.
+```
```

Comment:
@fsimonis are we actually still raising this error? I lost a bit track of all the back and forth here. Couldn't find the error message anymore when looking for it in the code.

Reply:
After a long discussion, we didn't merge this error.

https://github.com/precice/precice/pull/1867

Reply:
But we got a warning:

```
WARN: preCICE just returned a maximum time step size of {}. Such a small value can happen if you use many substeps per time window over multiple time windows due to added-up differences of machine precision
```